torch/lib/c10d/GlooDeviceFactory.cpp

```diff
@@ -36,16 +36,16 @@ C10_DEFINE_SHARED_REGISTRY_WITHOUT_WARNING(
 
 #if GLOO_HAVE_TRANSPORT_TCP
 static std::shared_ptr<::gloo::transport::Device> makeTCPDevice(
-    const std::string& interface,
+    const std::string& interface_name,
     const std::string& hostname) {
   TORCH_CHECK(
-      !interface.empty() || !hostname.empty(),
+      !interface_name.empty() || !hostname.empty(),
```

Comment:
convention in this file use camel instead of snake names (e.g., `glooDeviceTransport `), let keep consistent.